### PR TITLE
Remove Accounts-V1

### PR DESCRIPTION
**Pull request by rauljordan**
This is part of #7515, as we are now removing accounts-v1 from Prysm, encouraging users to use https://docs.prylabs.network/docs/wallet/ethdo.

To migrate into accounts-v2, please see https://docs.prylabs.network/docs/wallet/ethdo. Flags removed include
--disable-accounts-v2
--keystore-path
--keymanager
--unencrypted-keys
--password
--source-dirs
--target-dir
--keymanageropts
--passwords-dir

### CODE REVIEW

File: validator/client/metrics.go [hunk 1]
```diff
@@ -133,11 +132,7 @@ func (v *validator) LogValidatorGainsAndLosses(ctx context.Context, slot uint64)
 
 	var pks [][48]byte
 	var err error
-	if featureconfig.Get().EnableAccountsV2 {
-		pks, err = v.keyManagerV2.FetchValidatingPublicKeys(ctx)
-	} else {
-		pks, err = v.keyManager.FetchValidatingKeys()
-	}
+	pks, err = v.keyManagerV2.FetchValidatingPublicKeys(ctx)
```
Comment (0xKiwi) [inline validator/client/metrics.go:135]: Will there be another PR to remove mentions of V2?

**Review by prestonvanloon — APPROVED** [review]
LGTM. Please amend the PR description with migration instructions for beta.0 release notes.

**Review by 0xKiwi — APPROVED** [review]
